locust/env.py

```diff
@@ -115,6 +116,21 @@ def create_worker_runner(self, master_host, master_port):
             master_port=master_port,
         )
     
+    def apply_tags(self, include_tags=None, exclude_tags=None):
```

Comment:
I wonder if we should add `include_tags` and `exclude_tags` as instance variables (and constructor arguments) of `Environment`, and automatically applying the filtering in the constructor? Also, maybe `filter_tasks_by_tags` is a better name also for this method?